modules/src/address.rs

```diff
@@ -6,11 +6,16 @@ use anomaly::{BoxError, Context};
 use bech32::ToBase32;
 use bech32::{FromBase32, Variant};
 
+use std::str::FromStr;
 use tendermint::account::Id as AccountId;
 
-pub fn account_to_string(addr: AccountId) -> Result<String, BoxError> {
-    Ok(bech32::encode("cosmos", addr.to_base32(), Variant::Bech32)
-        .map_err(|e| Context::new("cannot generate bech32 account", Some(e.into())))?)
+pub fn encode_to_bech32(addr: String, hrp: String) -> Result<String, BoxError> {
+    let account =
+        AccountId::from_str(addr.as_str()).map_err(|e| Context::new("bad address", Some(e)))?;
+    Ok(
+        bech32::encode(hrp.as_str(), account.to_base32(), Variant::Bech32)
+            .map_err(|e| Context::new("cannot generate bech32 account", Some(e.into())))?,
+    )
```

Comment:
We should remove this from `ibc`, and remove the `bech32` dep. This should be in the cosmos chain impl.

Reply:
OK, thanks, I'll look into that.